### PR TITLE
[Az.App] Update-AzContainerApp examples

### PR DESCRIPTION
**Pull request by lijinpei2008**
## Description
Update cmdlet examples
 - Update-AzContainerApp

https://github.com/Azure/autorest.powershell/issues/978

## Checklist

- [ ] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from the scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** change version of module in pull request

### CODE REVIEW

File: src/App/examples/Update-AzContainerApp.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 ### Example 1: Update a Container App.
 ```powershell
-Update-AzContainerApp -Name azps-containerapp -ResourceGroupName azpstest_gp -Location canadacentral -DaprEnabled -DaprAppProtocol 'http' -DaprAppId "container-app-1" -DaprAppPort 8080
+$secretObject = Get-AzContainerAppSecret -ContainerAppName azps-containerapp -ResourceGroupName azpstest_gp
+$newSecretObject = @(0..($secretObject.Count-1))
+[array]::copy($secretObject,$newSecretObject,$secretObject.Count)
+$secretObject += New-AzContainerAppSecretsecretObject -Name "yourkey" -Value "yourvalue"
```
Comment (BethanyZhou) [inline src/App/examples/Update-AzContainerApp.md:6]: The analysis is correct, please correct New-AzContainerAppSecretsecretObject -> New-AzContainerAppSecretObject

**Comment by BethanyZhou** [comment]
Other issues are reported randomly, please remove ExampleIssues.csv. We can pass checker by re-running pipelines.

**Comment by BethanyZhou** [comment]
Why Get-AzContainerAppSecret doesn't contain suffix `Object` while we name its responding new cmdlet as New-AzContainerAppSecretObject

File: src/App/examples/Update-AzContainerApp.md (hunk 1 above)
Comment (BethanyZhou) [reply src/App/examples/Update-AzContainerApp.md]: Please correct this typo in generation branch as well

**Comment by lijinpei2008** [comment]
> Why Get-AzContainerAppSecret doesn't contain suffix `Object` while we name its responding new cmdlet as New-AzContainerAppSecretObject

The Get-AzContainerAppSecret cmdlet it has its own API, but New-AzContainerAppSecretObject don't has.
And the New cmdlet just to create an in-memory object, is obtained by expanding a parameter in the Update-AzContainerApp

**Review by BethanyZhou — APPROVED** [review]

**Pull request merged**